Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -114,21 +114,39 @@ struct HitData {
                  x, y, z, t);
 };
 
+struct MeasurementData {
+  /// Event-unique measurement identifier. Each value can appear at most once.
+  uint64_t hit_id;
+  /// Hit surface identifier.
+  uint64_t geometry_id = 0u;
+  /// Partially decoded hit surface identifier components.
+  uint32_t volume_id, layer_id, module_id;
+  /// Local hit information - bit identification what's measured
+  uint8_t local_key;
+  float local0, local1, phi, theta, time;
+  float varLocal0, varLocal1, varPhi, varTheta, varTime;
```

Comment:
```suggestion
  float var_local0, var_local1, var_phi, var_theta, var_time;
```
Consistency with the rest of the CSV files.

Reply:
Yup, done.